src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -98,9 +98,15 @@ public void setReplay(Replay replay) {
         .average()
         .ifPresent(averageRating -> ratingLabel.setText(i18n.number((int) averageRating)));
 
-    durationLabel.setText(Optional.ofNullable(replay.getEndTime())
-        .map(endTime -> timeService.shortDuration(Duration.between(replay.getStartTime(), endTime)))
-        .orElse(i18n.get("notAvailable")));
+    Integer replayTicks = replay.getReplayTicks();
+    if (replayTicks != null) {
+      durationLabel.setText(timeService.shortDuration(Duration.ofMillis(replayTicks * 100)));
```

Comment:
isn't millis 1000?

Reply:
Yes but there are 10 ticks per second. So `10/s * 100 = 1000/s`

Reply:
Oh! Math hidden!! :-) how about to show it as constant with explanation? Otherwise it's not visible on first sight